src/routes/Board/Board.tsx

```diff
@@ -22,7 +22,7 @@ function Board(props: BoardProps) {
         }
     }, [ props.match.params.id ]);
 
-    const state = useSelector((state: ApplicationState) => ({
+    const state:any = useSelector((state: ApplicationState) => ({
```

Comment:
We should have a look into this and replace the `any` with something meaningful (...if it is even required)